packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -184,10 +184,10 @@ const designSystemDefaults: DesignSystem = {
     neutralFillActiveDelta: 2,
     neutralFillSelectedDelta: 8,
 
-    neutralFillInputRestDelta: 4,
-    neutralFillInputHoverDelta: 4,
-    neutralFillInputActiveDelta: 4,
-    neutralFillInputSelectedDelta: 4,

```

Comment:
You want to title the PR as a "fix" and not a "chore" because "chore" signals to lerna that the change shouldn't introduce a version change (meaning this change wouldn't get published).